src/community/oseo/oseo-service/src/main/java/org/geoserver/opensearch/eo/response/TemplatesProcessor.java

```diff
@@ -158,6 +168,29 @@ protected void addUtilityFunctions(String baseURL, Map<String, Object> model) {
                                 throw new RuntimeException("Failed to encode geometry", e);
                             }
                         });
+        model.put("parseJSON", (TemplateMethodModelEx) this::parseJSON);
```

Comment:
```suggestion
        model.put("loadJSON", (TemplateMethodModelEx) this::parseJSON);
```

Reply:
model.put("loadJSON", parseJSON()); done like this casting and writing this:: is redundant and giving error.